src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -7,7 +7,9 @@
   },
   "frameworks": {
     "netstandard1.6": {
-      "NETStandard.Library": "1.5.0-rc3-24126-00"
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-rc3-24126-00"
```

Comment:
Can you please go ahead and update the version to match the latest in the repo? 24210-06 https://github.com/dotnet/cli/pull/3493